Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,55 @@
+# Microcontroller Signoff
+## FUNCTION OF THE SUBSYSTEM
+The microcontroller and its code are the centerpieces of the entire device.  They are responsible for operating, managing, communicating with, maintaining, and collaborating with all of the other subsystems and acts as both the brain and body of the device.  All the data and instructions must flow through the microcontroller at some point in its lifetime.  This gives a common place for all of the data to go and be readily available for whichever subsystem has need of it.  The hardware will be responsible for the actual actions and completion of any given task.  The code will be responsible for controlling the hardware and manipulating data.  This subsystem will be where most of the coding for the project is done as the operating code will be the largest by far.  The code itself will be where the “magic” happens as it allows for interfacing with the hardware and makes it more simple to utilize.
```

Comment:
Too much prose. Get to the point: state what this subsystem is expected to do. An engineer will read this, so avoid saying things like "this is where the magic happens."